web/src/main/java/org/fao/geonet/monitor/link/LinkMonitorInterface.java

```diff
@@ -0,0 +1,44 @@
+package org.fao.geonet.monitor.link;
+
+import jeeves.interfaces.Logger;
+import jeeves.server.resources.ResourceManager;
+import org.fao.geonet.GeonetContext;
+
+public class LinkMonitorInterface {
```

Comment:
I dunno about this one.  It's *probably* ok,  but if it's actually got implementation in it, then I think `LinkMonitorAbstract` would be preferable.  (think I said `AbstractLinkMonitor` above, but your name is more consistent).